### PR TITLE
Add Turkish localization for auto-scaling

### PR DESCRIPTION
**Pull request by muhsin09**
### Describe your changes

Google Docs: https://docs.google.com/document/d/147GDzT3SSOHAshambgUtci9rgRdHvpzXetQogjP7D7g/edit#heading=h.mo94ylk0tzfj

😎 Deploy Preview	https://deploy-preview-3078--cncfglossary.netlify.app/tr/auto-scaling/

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by muhsin09** [comment]
lang/tr

**Comment by muhsin09** [comment]
lang/tr

File: content/tr/auto-scaling.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+Daha önce altyapı ve uygulamalar, yoğun sistem kullanımını dikkate alacak şekilde tasarlanıyordu. Bu mimari, daha fazla kaynağın gereğinden az kullanıldığı ve değişen tüketici talebine karşı esnek olmadığı anlamına geliyordu. Bu esneklik eksikliği, işletmeye daha yüksek maliyetler ve talep fazlasından kaynaklanan kesintiler nedeniyle iş kaybı anlamına geliyordu.
+
+Bulut teknolojisi, uygulamaları ve bağımlılıklarını [sanallaştırma](/virtualization/) ve [konteynerleştirme](/containerization/) yoluyla ölçeklenebilir uygulamalar oluşturarak, organizasyonlara kullanıcı taleplerine göre ölçeklenebilir uygulamalar inşa etme olanağı sağlar. Uygulama taleplerini izleyip otomatik olarak ölçeklenerek, optimal bir kullanıcı deneyimi sunabilirler. Netflix'in her Cuma akşamı yaşadığı izleyici artışını düşünün. Otomatik ölçekleme, dinamik olarak daha fazla kaynak eklemeyi ifade eder: örneğin, daha fazla video akışına izin veren sunucu sayısını artırma ve tüketim normalleştikten sonra ölçeği küçültme.
```
Comment (rwxdash) [inline content/tr/auto-scaling.md:12]: Could you update the links to include the `/tr/` prefix, so that they link to the Turkish content?

**Review by rwxdash — APPROVED** [review]

File: content/tr/auto-scaling.md (hunk 1 above)
Comment (muhsin09) [reply content/tr/auto-scaling.md]: updated and pushed.

**Review by aliok — APPROVED** [review]

**Pull request merged**